src/components/NewCommentForm.tsx

```diff
@@ -75,25 +133,48 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames(
+              'textarea', { 'is-danger': errorTextComment },
+            )}
+            value={comment}
+            onChange={(e) => setComment(e.target.value)}
           />
         </div>
 
-        <p className="help is-danger" data-cy="ErrorMessage">
-          Enter some text
-        </p>
+        {errorTextComment && (
+          <p className="help is-danger" data-cy="ErrorMessage">
+            Enter some text
+          </p>
+        )}
       </div>
 
       <div className="field is-grouped">
         <div className="control">
-          <button type="submit" className="button is-link is-loading">
+          <button
+            type="submit"
+            className={classNames(
+              'button is-link', {
+                'is-loading': isLoading && !isResponse,
+              },
+            )}
+            // onClick={() => setIsLoading(true)}
+          >
             Add
           </button>
         </div>
 
         <div className="control">
           {/* eslint-disable-next-line react/button-has-type */}
-          <button type="reset" className="button is-link is-light">
+          <button
+            type="reset"
+            className="button is-link is-light"
+            onClick={() => {
+              setAuthor('');
+              setEmail('');
+              setComment('');
+              setIsSubmit(false);
+            }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)